### PR TITLE
Add modpack page warning + Fix notifications nav titles

### PR DESCRIPTION
**Pull request by venashial**
image

### CODE REVIEW

File: nuxt.config.js [hunk 1]
```diff
@@ -228,7 +228,7 @@ export default {
    ** See https://axios.nuxtjs.org/options
    */
   axios: {
-    baseURL: 'https://api.modrinth.com/v2/',
+    baseURL: 'https://staging-api.modrinth.com/v2/',
```
Comment (wafflecoffee) [inline nuxt.config.js:231]: Staging

File: pages/notifications.vue [hunk 2]
```diff
@@ -104,14 +104,18 @@ export default {
...
+        (it) => it.type !== null
+      )) {
         obj[
           Object.keys(NOTIFICATION_TYPES).find(
             (key) => NOTIFICATION_TYPES[key] === notification.type
           )
         ] = true
       }
 
+      console.log(obj)
```
Comment (wafflecoffee) [inline pages/notifications.vue:117]: Console log

**Review by wafflecoffee — CHANGES_REQUESTED** [review]

File: nuxt.config.js (hunk 1 above)
Comment (venashial) [reply nuxt.config.js:231]: It should use staging in development, I accidentally left it with the production URL in the the last PR

File: pages/notifications.vue (hunk 2 above)
Comment (venashial) [reply pages/notifications.vue:117]: Whoops

File: nuxt.config.js (hunk 1 above)
Comment (wafflecoffee) [reply nuxt.config.js:231]: Vercel builds directly from source, though

Comment (venashial) [reply nuxt.config.js:231]: Vercel & other production environments have an environment variable that override this

**Review by wafflecoffee — APPROVED** [review]

**Pull request merged**